stanford/lagunita/edxapp1.yml

```diff
@@ -5,3 +5,4 @@
     CLUSTER_EBS_SIZE: 64
     CLUSTER_INSTANCE_TYPE: 'm4.4xlarge'
     COMMON_DEPLOYMENT: lagunita
+    EDXAPP_FEATURES_ALLOW_COURSE_RERUNS: false
```

Comment:
But disable for `edxapp1`

and implicitly for `edxapp2` (since it includes/extends this file).